cocos/spine/skeleton.ts

```diff
@@ -773,8 +773,8 @@ export class Skeleton extends UIRenderer {
         this._textures = skeletonData.textures;
 
         this._refreshInspector();
-        if (this.defaultAnimation) this.animation = this.defaultAnimation.toString();
         if (this.defaultSkin && this.defaultSkin !== '') this.setSkin(this.defaultSkin);
+        if (this.defaultAnimation) this.animation = this.defaultAnimation.toString();
```

Comment:
When adding a prefab asset to the node tree, the animation settings are applied before the skin settings. This causes the animation to not be set correctly, as the animation depends on the skin.

Reply:
It is better to add a comment in to codes saying it.